src/app/components/MainContent/index.jsx

```diff
@@ -31,7 +33,7 @@ const MainContent = ({ data }) => {
 MainContent.propTypes = {
```

Comment:
We should remove `data.model` and just pass `blocks` as props, which means changes [here](https://github.com/bbc/simorgh/pull/326/files#diff-89fcabca58d80c9d5cba58347724d5ceR28) and [here](https://github.com/bbc/simorgh/pull/326/files#diff-89fcabca58d80c9d5cba58347724d5ceR19), and tests/stories.